test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,41 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list (it has no null objects)
+        assertHasNoNull();
+
+        // there are null objects
+        assertHasNull((Object) null);
+        assertHasNull((Object) null, 1);
+        assertHasNull((Object) null, "");
+        assertHasNull((Object) null, "abc");
+        assertHasNull((Object) null, 1, "", "abc");
+
+        // there are no null objects
+        assertHasNoNull("abc", "ab", "a");
+        assertHasNoNull(1, 2);
+        assertHasNoNull(objectList);
+        assertHasNoNull(objectList, "def", "ghi", "jkl");
+
+        // some null objects
+        assertHasNoNull("abc", "abc");
+        assertHasNoNull("abc", "", "abc", "ABC");
+        assertHasNoNull("", "abc", "a", "abc");
+        assertHasNoNull(1, Integer.valueOf(1));
+        assertHasNull(null, 1, Integer.valueOf(1));
```

Comment:
It'll be more readable to add a blank line between the assertHasNoNull and assertHasNull checks